zerver/openapi/zulip.yaml

```diff
@@ -7308,13 +7321,19 @@ paths:
                               description: |
                                 The topic of the message immediately
                                 after this edit event.
+
+                                If `allow_empty_topic_name` parameter is `false`,
+                                the `""` value is replaced with `"general chat"`.
                             prev_topic:
                               type: string
                               description: |
                                 Only present if message's topic was edited.
 
                                 The topic of the message immediately
                                 prior to this edit event.
+
+                                If `allow_empty_topic_name` parameter is `false`,
+                                the `""` value is replaced with `"general chat"`.
```

Comment:
I don't think we need this repetition here; the behavior is explained clearly in the description of the parameter.

Reply:
Removed